frontend/packages/dev-console/src/components/health-checks/create-health-checks-probe-utils.ts

```diff
@@ -6,8 +6,8 @@ import { healthChecksDefaultValues } from './health-checks-probe-utils';
 
 export const constructProbeData = (data: HealthCheckProbeData, resourceType?: Resources) => {
   const probeData = {
-    ...(data.failureThreshold && { failureThreshold: data.failureThreshold }),
-    ...(data.successThreshold && { successThreshold: data.successThreshold }),
+    ...(data.failureThreshold && { failureThreshold: _.toNumber(data.failureThreshold) }),
+    ...(data.successThreshold && { successThreshold: _.toNumber(data.successThreshold) }),
```

Comment:
Can we use `_.toInteger` here? This was already used in this utils function and we know that the data value only accepts positive integers here.